geth/api/api.go

```diff
@@ -248,3 +248,8 @@ func (api *StatusAPI) AppStateChange(state string) {
 	}
 	api.b.AppStateChange(appState)
 }
+
+// StatusGoVersion returns build version
+func (api *StatusAPI) StatusGoVersion() string {
```

Comment:
I think it can be named `Version()`. There are no other components than `StatusGo`.